docs/source/apidoc.rst

```diff
@@ -26,6 +26,11 @@ GHZ Circuits
 .. automodule:: mitiq.benchmarks.ghz_circuits
    :members:
 
+Volume Circuits
```

Comment:
I suggest using the more standard "Quantum Volume Circuits" instead of just "Volume Circuits" , as it has a quite specific meaning.